requirements-dev.txt

```diff
@@ -12,8 +12,6 @@ asv==0.6.5
     # via -r requirements-dev.in
 asv-runner==0.2.1
     # via asv
-backports-asyncio-runner==1.2.0
```

Comment:
Unrelated, but pinning the exact version of `backports-asyncio-runner` was preventing installing dev deps on python 3.12